tests/checker/good/gold/dead_code_test10.gold

```diff
@@ -0,0 +1,84 @@
+{
+  "cashflow_tags": { "State variables": [], "ADT constructors": [] },
+  "contract_info": {
+    "scilla_major_version": "0",
+    "vname": "Dead",
+    "params": [],
+    "fields": [],
+    "transitions": [ { "vname": "Foo", "params": [] } ],
+    "procedures": [
+      {
+        "vname": "Bar",
+        "params": [ { "vname": "a", "type": "dead_code_test11.A" } ]
```

Comment:
`dead_code_test11` here looks strange, since the file name is `dead_code_test10.gold`.
Also, the file name convention is `name.scilla.gold`

Reply:
That's right, it was an artifact from my previous experiments. Sorry for the confusion.

Reply:
no worries :)